src/main.zig

```diff
@@ -2688,6 +2688,15 @@ fn buildOutputType(
         lib: Compilation.SystemLib,
     }) = .{};
 
+    var libc_installation: ?LibCInstallation = null;
+    defer if (libc_installation) |*l| l.deinit(gpa);
+
+    if (libc_paths_file) |paths_file| {
+        libc_installation = LibCInstallation.parse(gpa, paths_file, cross_target) catch |err| {
```

Comment:
```suggestion
    var libc_installation: ?LibCInstallation = null;

    if (libc_paths_file) |paths_file| {
        libc_installation = LibCInstallation.parse(arena, paths_file, cross_target) catch |err| {
```

This function is rather long; using `arena` rather than `gpa` will avoid the code bloat of duplicating that unnecessary cleanup code at every `try` in the function body below.

Don't forget to change the call to `findNative` below too.

Reply:
Ah, I hadn't thought of that, good to know!